client/pages/edit-event.jsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import DatePicker from 'react-datepicker';
+
+export default class EditEvent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      event: {
+        title: '',
+        date: '',
+        startingtime: '',
+        address: '',
+        city: '',
+        state: '',
+        mapLocation: { }
+      },
+      showModal: false
+    };
+    this.handleUpdate = this.handleUpdate.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleDate = this.handleDate.bind(this);
+    this.handleAddress = this.handleAddress.bind(this);
+    this.handleCity = this.handleCity.bind(this);
+    this.handleState = this.handleState.bind(this);
+  }
+
+  componentDidMount() {
+    fetch(`/api/events/${this.props.eventID}`)
+      .then(res => res.json())
+      .then(data => {
+        this.setState({ event: data });
+      })
+      .catch(err => {
+        console.error('There is a error', err);
+      });
```

Comment:
A normal user won't have the console open, so I'd suggest you also add an `alert` or something to your `.catch` handlers so they're not just sitting there waiting.